src/tox_current_env/hooks.py

```diff
@@ -72,7 +72,9 @@ def tox_configure(config):
     if _plugin_active(config.option):
         config.skipsdist = True
         for testenv in config.envconfigs:
-            config.envconfigs[testenv].whitelist_externals = "*"
+            for entry in ['allowlist_externals', 'whitelist_externals']:
+                if getattr(config.envconfigs[testenv], entry, None):
+                    setattr(config.envconfigs[testenv], entry, "*")
```

Comment:
The thing I don't understand is how does this actually work. We can set both as long as they are the same value?

Reply:
You cannot set both. Tox will raise an error if both options are present inside the same file, regardless their location. The old option was kept for compatibility but as soon you start using the new one, you must replace old ones with new one everywhere.  I know this because I had the same trick to apply to another plugin I created.

Reply:
And when I read this code it appears as if it sets both. What am I not getting?

Reply:
Let me summarize this:

 - If `allowlist_externals` is set to any value, we set it to `*` instead.
 - If `whitelist_externals` is set to any value, we set it to `*` instead.
 - If both are set, we set both, but the configuration was already invalid, so no harm was done.
 - If none was set, what happens?